test/upgrade/prober/verify.go

```diff
@@ -41,11 +41,15 @@ const (
 	jobWaitTimeout  = 5 * time.Minute
 )
 
-// Verify will verify prober state after finished has been send.
+// Verify will verify prober state after finished has been sent.
 func (p *prober) Verify(ctx context.Context) ([]error, int) {
 	report := p.fetchReport(ctx)
-	p.log.Infof("Fetched receiver report. Events propagated: %v. "+
-		"State: %v", report.Events, report.State)
+	availRate := 0.0
+	if report.TotalRequests != 0 {
+		availRate = float64(report.EventsSent*100) / float64(report.TotalRequests)
+	}
+	p.log.Infof("Fetched receiver report. Events propagated: %v. State: %v.  Total event send requests: %v, avail rate %.3f%%.",
+		report.EventsSent, report.State, report.TotalRequests, availRate)
```

Comment:
Slight reformatting, and using full words instead of abbreviations.

```suggestion
	p.log.Infof("Fetched receiver report. Events propagated: %v. State: %v.",
		report.EventsSent, report.State)
	p.log.Infof("Availability: %.3f%%, Requests sent: %d.",
		availRate, report.TotalRequests)
```

Reply:
Fixed as suggested.